quarto/config/routes.yaml

```diff
@@ -26,3 +26,38 @@ game_place:
         idGame: '\d+'
         x: '\d+'
         y: '\d+'
+
+gameapi_new:
+  path: /gameapi
+  controller: 'App\Controller\GameApiController::new'
+
+gameapi_list:
+  path: /gameapi/list
+  controller: 'App\Controller\GameApiController::list'
+
+gameapi:
+  path: /gameapi/{idGame}
+  controller: 'App\Controller\GameApiController::current'
+  requirements:
+        idGame: '\d+'
+
+gameapi_select:
+  path: /gameapi/{idGame}/select/{piece}
+  controller: 'App\Controller\GameApiController::select'
+  requirements:
+        idGame: '\d+'
+        piece: '\d+'
+
+gameapi_place:
+  path: /gameapi/{idGame}/place/{x}/{y}
+  controller: 'App\Controller\GameApiController::place'
+  requirements:
+        idGame: '\d+'
+        x: '\d+'
+        y: '\d+'
+
+all:
```

Comment:
There should be better way to [catch 404](https://symfony.com/doc/current/controller/error_pages.html). :)